src/pages/home/Home.tsx

```diff
@@ -1,5 +1,45 @@
-import { FC } from "react";
+import { PostCard } from "components";
+import { Post } from "Interfaces";
+import { FC, useEffect } from "react";
+import { Grid } from "react-loader-spinner";
+import { useDispatch, useSelector } from "react-redux";
+import { useAxios } from "utils";
+import { RootState } from "store";
+import "./home.css";
+import { timelinePosts } from "reducers/postsSlice";
 
 export const Home: FC = () => {
-  return <div>Home</div>;
+  const timeline = useSelector((state: RootState) => state.posts.timelinePosts);
+  const { loading, operation } = useAxios();
+  const dispatch = useDispatch();
+
+  useEffect(() => {
```

Comment:
You need to add user dependency as this page is does not reflect user after login in.

Reply:
Done. Thanks for pointing it out.